addons/google_calendar/utils/google_calendar.py

```diff
@@ -28,6 +28,22 @@ class GoogleCalendarService():
     def __init__(self, google_service):
         self.google_service = google_service
 
+    @requires_auth_token
+    def get_event(self, eventId, sync_token, token=None, timeout=TIMEOUT):
+        url = f"/calendar/v3/calendars/primary/events/{eventId}"
+        headers = {'Content-type': 'application/json'}
+        params = {'access_token': token, 'syncToken': sync_token}
```

Comment:
Note that "get_events" adds time boundaries ... why not doing the same, any reason ? Note that extracting code would help for spotting differences :D